### PR TITLE
Add the ability to update all dependencies for a repo

### PR DESCRIPTION
**Pull request by quantumew**
@aarongodin Thanks for the review on these! Let me know what you think about this one. Just a new subcommand, `turnup update-all`, that eventually just runs `npm update --package-lock-only`.

### CODE REVIEW

File: src/actions/update.js [hunk 1]
```diff
@@ -31,29 +66,47 @@ const updateRepository = async (adapter, repo, packageName, packageVersion, opti
     packageDef.dependencies[packageName] = packageVersion
   }
 
+  commitUpdates(ACTION, adapter, options, repository, branchName, packageDef, packages.lockfile.create)
+}
+
+const updateRepositoryAll = async (adapter, repository, options) => {
+  notify(ALL_ACTION, `Running npm update against ${chalk.italic(repository.fullName)}`)
+  const branchName = `turnup/update-all`
```
Comment (aarongodin) [inline src/actions/update.js]: To make sure the user can run this again and not encounter an issue with existing branch name, let's add some kind of unique value here. Perhaps bring in the `uuid` library and use `v4()`, slicing off the first six characters.

Comment (aarongodin) [reply src/actions/update.js]: And appending to `turnup/update-all`.

**Review by aarongodin — CHANGES_REQUESTED** [review]

**Comment by aarongodin** [comment]
I added you as a contributor on the repo if you'd like to just branch here in the future rather than your fork (makes it easier to checkout your changes locally).

File: src/actions/update.js (hunk 1 above)
Comment (quantumew) [reply src/actions/update.js]: Good call e45c662ab012952d851ab5b221752215ce1272af

**Review by aarongodin — APPROVED** [review]

**Pull request merged**